src/App.tsx

```diff
@@ -15,48 +17,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortField,
+  reverse = false,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortField.SORT_FIELD_LENGTH:
+          return good1.length - good2.length;
+
+        case SortField.SORT_FIELD_ALPHABET:
+          return good1.localeCompare(good2);
+
+        default:
+          return 0;
```

Comment:
```suggestion
          return goods;
```

Reply:
> 

It is allowed in Ts to use 0 to compare strings when you need to return an unchanged array, isn't it?